exercises/practice/pascals-triangle/.meta/config.json

```diff
@@ -1,6 +1,24 @@
 {
   "blurb": "Compute Pascal's triangle up to a given number of rows.",
-  "authors": [],
+  "authors": [
+    "deathsec"
```

Comment:
Hmm... I'm pretty sure that's my old username.. Well anyways I contributed a long back I guess the author username correctness is not that important in this context.

Reply:
@antony-jr If you'd like the contributions to "count" in Exercism for yourself (ie show your face by the exercises and award you Reputation in the coming release), then you can send a PR to this PR to fix that :)

Reply:
@iHiD no worries, consider this resolved. I'm just chiming in after a long time. :smile:

Reply:
Totally. But that's why we tagged you! I want to make sure everyone gets the recognition for the hard-work they've done in the past. And Reputation plays a big part in v3 so I don't want anyone to miss out :)